app/quadraticlands/models.py

```diff
@@ -198,3 +206,255 @@ class SchwagCoupon(models.Model):
     def __str__(self):
         """String for representing the Model object."""
         return f'{self.discount_type}, {self.coupon_code}, {self.profile}'
+
+def create_game_helper(handle, title):
+    game = Game.objects.create(
+        _type = 'diplomacy',
+        title = title,
+        )
+    player = GamePlayer.objects.create(
+        game = game,
+        profile = Profile.objects.get(handle=handle),
+        active = True,
+        admin = True
+        )
+    game.players.add(player)
+
+    GameFeed.objects.create(
+            game=game,
+            player=player,
+            data={
+                'action': 'created the game',
+            },
+            )
+    game.add_player(player.profile.handle)
+    return game
+
+class Game(SuperModel):
```

Comment:
need to cache the common data attributes for the QD.  its too slow wiht 12 players now

Reply:
this is done now